build/vuejs-quiz-qpp/quiz.json

```diff
@@ -0,0 +1,258 @@
+[
+    {
+        "Question":
+          "In CSS, Flex is a shorthand property for which of the following properties?",
+        "Answer": "flex-grow, flex-shrink, and flex-basis",
+        "Distractor1": "justify-content, align-items, and align-content",
+        "Distractor2": "flex-direction, flex-wrap, and flex-flow",
+        "Distractor3": "flex-direction, order, and flex-flow",
+        "Explanation":
+          "Flex is a shorthand property for all three of these properties: flex-grow, flex-shrink, and flex-basis.",
+        "Link": "https://developer.mozilla.org/en-US/docs/Web/CSS/flex"
+      },
+      {
+        "Question":
```

Comment:
This comment applies to all quiz questions. 

In your prototype [here](https://learnvue-quizapp.netlify.app/), the user can choose the difficulty of questions before the quiz starts. 

We should add a field called Difficulty that only allows the options of easy, medium, hard, or mixed